modules/ROOT/pages/database-administration/standard-databases/delete-databases.adoc

```diff
@@ -103,4 +103,46 @@ An example could look like this:
 DROP DATABASE customers IF EXISTS DUMP DATA
 ----
 
+[[delete-databases-with-aliases]]
+=== Delete a database with local database aliases targeting it
+
+In general, deleting a database that is the target of any local database aliases will fail due to those database aliases.
+Any remote database aliases targeting the database does not affect the deletion of the database.
+
+By appending `CASCADE ALIASES` to the command `DROP DATABASE`, those local database aliases can be dropped together with the database.
+Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behaviour as if you create them targeting a non-existing database).
```

Comment:
```suggestion
Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behaviour as if they are created targeting a non-existing database).
```
Again US / UK spelling of behaviour, docs team probably can confirm.

Reply:
```suggestion
Any remote database aliases targeting the database will not be affected of the `CASCADE ALIASES`, they will simply no longer resolve their targets (which is the same behavior as if they are created targeting a non-existing database).
```